aio_pika/patterns/rpc.py

```diff
@@ -288,38 +286,34 @@ async def on_call_message(
 
         await message.ack()
 
-    def serialize(self, data: Any) -> bytes:
-        """ Serialize data to the bytes.
-        Uses `pickle` by default.
-        You should overlap this method when you want to change serializer
-
-        :param data: Data which will be serialized
-        :returns: bytes
-        """
-        return super().serialize(data)
-
-    def deserialize(self, data: bytes) -> Any:
-        """ Deserialize data from bytes.
-        Uses `pickle` by default.
-        You should overlap this method when you want to change serializer
-
-        :param data: Data which will be deserialized
-        :returns: :class:`Any`
-        """
-        return super().deserialize(data)
-
-    def serialize_exception(self, exception: Exception) -> bytes:
-        """ Serialize python exception to bytes
-
-        :param exception: :class:`Exception`
-        :return: bytes
-        """
-        return pickle.dumps(exception)
+    def serialize_exception(self, exception: Exception) -> Any:
+        """ Serialize python exception to bytes """
```

Comment:
Due to the name of the method and due to the docstring says it serializes exception to bytes it's expected that it wouldn't return exception as is but there is no serialization and it returns `Any` type..

Reply:
Cause for the JsonRPC it's a dictionary. This method serialises an exception, not returning an exception.

Reply:
Then "to bytes" needs to be removed from docstring, I guess.